tests/unit-tests/test-class-sensei-course-enrolment-provider-results.php

```diff
@@ -16,19 +16,19 @@ public function setup() {
 	 */
 	public function testFromJson() {
 		$base = [
-			't' => microtime( true ),
+			't' => microtime( true ) - 4,
```

Comment:
Why subtract 4?

Reply:
1 would probably do, but I really just want to make sure that a new `microtime()` was never set in the results constructor (with `assertEquals` having a delta of `0.1`)